backend/src/libs/interfaces/service.interface.ts

```diff
@@ -1,5 +1,6 @@
 interface IService<T = unknown> {
-  find(payload: unknown): Promise<T>;
+  find(id: number): Promise<T>;
+  findByEmail(email: string): Promise<T>;
```

Comment:
i think you should use id: unknown and email: unknown , same as  in repository interface

Reply:
Why? we know types of email and ID